src/auslib/blobs/schemas/gmp.yml

```diff
@@ -69,3 +69,9 @@
                           type: string
                           description: Where the client can download the file for this vendor+platform combination.
                           format: uri
+                        mirrorUrls:
```

Comment:
It's a little bit unfortunate that we have two separate places where packages can exist. In an ideal world, I imagine we would've put everything under `mirrorUrls` and required it to have one or more entry. 

Obviously doing so would not be backwards compatible though - can you add a comment to help us remember why we ended up this way for posterity?

(I guess technically we could do this on the input side, and still serve the a compatible XML...but then we'd have to rev the version on the GMP blob as well...it's unlikely to be worth the effort.)

Reply:
Commented!